room/room-compiler-processing/src/main/java/androidx/room/compiler/processing/ksp/KspTypeMapper.kt

```diff
@@ -81,7 +81,7 @@ object KspTypeMapper {
         mapping["java.util.List"] = "kotlin.collections.MutableList"
         mapping["java.util.ListIterator"] = "kotlin.collections.ListIterator"
         mapping["java.util.Map"] = "kotlin.collections.MutableMap"
-        mapping["java.util.Map.Entry"] = "Map.kotlin.collections.MutableEntry"
```

Comment:
@yigit I noticed this seemed like a preexisting bug that you might want fixed

Reply:
:facepalm: thanks!